build/Dockerfile

```diff
@@ -1,17 +1,15 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.14 AS builder
+FROM registry.hub.docker.com/library/golang:1.14 AS builder
```

Comment:
I think we don't want the image changes here, right?

Reply:
I can easily drop that one commit. Are there any others that we should avoid merging? Is there a list somewhere?

Reply:
I don't think there's a list. In the past we have either taken everything, even if we don't use some of it (such as the tilt work) because that makes future merges easier, or we have cherry-picked individual PRs. Cherry picking is usually how we get into a state where it's hard to do a proper merge, though, so I would rather we avoid that if we can.

The one other thing in this set of changes that made me hesitate was the API change to add the error count, but that's a feature we wanted downstream so we should take it.